package.json

```diff
@@ -15,11 +15,11 @@
     "node-fetch": "^3.3.2",
     "react": "^16.8.1",
     "react-dom": "^16.8.1",
-    "react-object-inspector": "^0.2.1"
+    "react-inspector": "^6.0.2"
```

Comment:
`react-object-inspector` is deprecated, points to using `react-inspector` instead.